src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeContactPage.java

```diff
@@ -0,0 +1,51 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeContactPage extends AbstractPage {
+
+    @FindBy(id = "id_contact")
+    private ExtendedWebElement subjectHeading;
+
+    @FindBy(id = "email")
+    private ExtendedWebElement emailAddress;
+
+    @FindBy(id = "id_order")
+    private ExtendedWebElement orderReference;
+
+    @FindBy(id = "message")
+    private ExtendedWebElement message;
+
+    @FindBy(id = "submitMessage")
+    private ExtendedWebElement send;
+
+    public AutomationPracticeContactPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void selectSubjectHeading(String subjectHeading) {
+        this.subjectHeading.select(subjectHeading);
```

Comment:
you can interact with elements without `this.` , just can be user as 
`subjectHeading.select(subjectHeading);`
update every element

Reply:
Ok, nice, so it is not a bad practice to avoid it, I will take it in count and fix it